src/io.rs

```diff
@@ -0,0 +1,375 @@
+// Rust Bitcoin Library
```

Comment:
I assume we should just use whatever lands in either https://github.com/rust-bitcoin/bitcoin_hashes/pull/126 or https://github.com/rust-bitcoin/bitcoin_hashes/pull/127 (and probably take some of the stuff that's here and copy it there).